DeleteConfirmationModal.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
We have a modal component here:

https://github.com/digitalbazaar/bedrock-quasar-components/blob/main/BrQModal.vue

Do we want to use that instead?

Reply:
^ @JoshDunnDev ?

Reply:
Might not be the best in the case. This delete modal is a modal within a modal. The original modal does use that modal component, but it may cause confusion if we use that same component for both modals here because they would completely overlap each other instead of partially.

Reply:
@JoshDunnDev,

If by "overlap" you're referring to visually -- why not just size them differently? My recommendation about reusing an existing component is to avoid having more than one thing to maintain -- and to avoid having to potentially make stylistic changes in multiple places. Managing a single component for modals provides consistency in the UI; if we have needs for different sizes our component should just support tweaks like that, IMO.

Reply:
@dlongley Yeah we should probably add additional support for custom sizing or color options in the future. I will further explore this later in this week and see if we can come up with a better solution here. I think we should go ahead and push through what Ganesh put together here for now.

Reply:
@JoshDunnDev, sounds good. Feel free to turn one of these comments into an issue for tracking if necessary.

Reply:
Issue has been filed: https://github.com/digitalbazaar/bedrock-vue-credential-card/issues/44